bgpd/bgp_nexthop.c

```diff
@@ -1111,34 +1113,42 @@ static void bgp_show_nexthops(struct vty *vty, struct bgp *bgp,
 		tree = &bgp->nexthop_cache_table;
 
 	if (afi == AFI_IP || afi == AFI_IP6) {
-		if (json)
-			json_afi = json_object_new_object();
+		if (uj)
+			vty_out(vty, "%s:{", (afi == AFI_IP) ? "\"ipv4\"" : "\"ipv6\"");
```

Comment:
afi2str_lower()

Reply:
> afi2str_lower()

The expected output is something like this **"ipv4":{"27.0.0.3"** where the ipv4 is within double quotation marks.
Hence we have used the escape sequence within the string.
But if we use api afi2str_lower(), it would display something like this **ipv4:{"27.0.0.3"**